docs/guide/apm-getting-started.asciidoc

```diff
@@ -2,22 +2,22 @@
 include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
 
 [[apm-gettting-started]]
-= Getting Started with Elastic APM
+= Getting Started with APM
 
 == Overview
 
 
-_Elastic APM_ is an application performance monitoring system built on the Elastic Stack.
```

Comment:
Should we make it a variable in case we change our mind again? :-)

Reply:
Don't think I was part of this discussion. In the context of elastic.co, we're calling it "APM", and it'll be named "APM" on the Download page

Reply:
I'm more joking because you had to replace it in quite a few places :-)

Reply:
:) we can revert those changes here and get back to it later.

Reply:
No all good, I like that we call it just APM.

Reply:
++